layouts/css/page-modules/_contributor-card.scss

```diff
@@ -12,15 +11,13 @@
   > a {
     height: 40px;
     width: 40px;
-    margin-right: 1em;
+    flex: 0 0 auto;
   }
 
   p {
+    padding-right: 1em;
```

Comment:
@MartijnCuppens maybe this was supposed to be `padding-left`?

Reply:
Yeah, indeed